src/lib/ibcclient.ts

```diff
@@ -957,8 +957,8 @@ export class IbcClient {
       [msg],
       calculateFee(this.limits.connectionHandshake, this.gasPrice)
```

Comment:
I guess using auto-gas is another issue?
I would love to see that. But I guess one thing at a time.